test/index.js

```diff
@@ -8,6 +8,9 @@ var Z = require ('sanctuary-type-classes');
 var doctest = require ('..');
 
 
+var esmSupported = Number ((process.versions.node.split ('.'))[0]) >= 9;
```

Comment:
Shall we rename the variable `esm` to match __bin/doctest__, or do you think a more descriptive name is warranted in this context?

Reply:
I prefer the more descriptive name. But we could update it in bin for consistency.

Reply:
:zap:

Reply:
Sounds good. I like to be able to verify that duplicated code is identical using <kbd>⌘</kbd><kbd>F</kbd>. :)